### PR TITLE
Fix the show-deployments command

### PR DESCRIPTION
**Pull request by alexwlchan**
I tried to run it in the root of the storage service, and I got an error:

```
Traceback (most recent call last):
  File "/usr/local/bin/weco-deploy", line 11, in <module>
    load_entry_point('weco-deploy', 'console_scripts', 'weco-deploy')()
  File "/Users/alexwlchan/repos/weco-deploy/src/deploy/deploy.py", line 350, in main
    cli()
  File "/Users/alexwlchan/Library/Python/3.8/lib/python/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/alexwlchan/Library/Python/3.8/lib/python/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/alexwlchan/Library/Python/3.8/lib/python/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/alexwlchan/Library/Python/3.8/lib/python/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/alexwlchan/Library/Python/3.8/lib/python/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/alexwlchan/Library/Python/3.8/lib/python/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/alexwlchan/repos/weco-deploy/src/deploy/deploy.py", line 328, in show_deployments
    summaries = _summarise_release_deployments(releases)
  File "/Users/alexwlchan/repos/weco-deploy/src/deploy/deploy.py", line 313, in _summarise_release_deployments
    'requested_by': deployment.get('requested_by').split('/')[-1],
AttributeError: 'dict' object has no attribute 'split'
```

The problem is that `deployment` is actually a dict `{"requested_by": "…", "id": "…"}`.

This patch fixes it, and adds pretty-printed tables and dates in the output:

Screenshot 2020-07-20 at 13 18 38

### CODE REVIEW

**Comment by alexwlchan** [comment]
Aaaaand now it’s broken again. Weird.

Ah, I wonder if old-style and new-style deployments are different?

**Comment by alexwlchan** [comment]
Think it's working now!

**Review by kenoir — APPROVED** [review]

**Comment by kenoir** [comment]
Very nice.

File: tests/test_pretty_printing.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import datetime
+
+import pytest
+
+from deploy.pretty_printing import pprint_date
+
+
+@pytest.mark.parametrize("date_obj, now, expected_str", [
```
Comment (kenoir) [inline tests/test_pretty_printing.py:8]: 👏

**Pull request merged**